Improvement: Fix some regex tests from old pr

not-a-cowfr opened the pull request:
## What
I really dont know if there was a better way to update this, just following what nopo said because nopo is good at this stuff 🙏
[old pr](https://github.com/hannibal002/SkyHanni/pull/2348)


## Changelog Improvements
+ Added more options to block messages. - not_a_cow
    * Chat Filter settings now include blocking parkour messages and teleport pad messages.

not-a-cowfr committed:
Improvement: Hide parkour messages

wasnt able to test, NOT SURE IF IT WORKS

not-a-cowfr committed:
Added additional messages

added additional messages, improved formatting

not-a-cowfr committed:
Added teleport pad messages option

unconfident that this works

not-a-cowfr committed:
Added regex tests (i think)

idk if i did this right
- added tests
- fixed typos (im very smart)

not-a-cowfr committed:
Fix regex tests

not-a-cowfr committed:
re-add hannibal's code cleanup

CalMWolfs commented on src/main/java/at/hannibal2/skyhanni/features/chat/ChatFilter.kt:
```diff
@@ -464,8 +460,8 @@ object ChatFilter {
         "achievement_get" to achievementGetPatterns,
-        "parkour" to parkourPatterns,
-        "teleport_pads" to teleportPadPatterns,
+        "hideParkour" to parkourPatterns,
+        "hideTeleportPads" to teleportPadPatterns,
```
everythign elese here has underscores so this camelCase seems out of place

not-a-cowfr replied:
just copied what hannibal changed, thought it was best to not go against that

https://github.com/hannibal002/SkyHanni/pull/2348/commits/473f962286372e0e09fea46a355e12c00fcd2589

not-a-cowfr committed:
small changes

still no clue what you want me to do with "use repo patterns and repo pattern lists"

not sure if im stupid or blind but i dont see why that is needed

not-a-cowfr committed:
Merge branch 'beta' into hide-parkour-messages

hannibal002 committed:
Merge branch 'beta' into fork/Not-a-cowfr/hide-parkour-messages

hannibal002 committed:
revert removing of regex patterns
